static/js/components/SiteSidebar.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { shallow } from "enzyme"
+
+import SiteSidebar from "./SiteSidebar"
+
+import { makeWebsite } from "../util/factories/websites"
+import { siteUrl, siteContentListingUrl } from "../lib/urls"
+
+import { Website } from "../types/websites"
+
+describe("SiteSidebar", () => {
+  let website: Website
+
+  beforeEach(() => {
+    website = makeWebsite()
+  })
+
+  it("renders some links", () => {
+    const wrapper = shallow(<SiteSidebar website={website} />)
+
+    const links = wrapper
+      .find("NavLink")
+      .map(link => [link.text(), link.prop("to")])
+
+    const expected = [
+      ["Content", siteUrl(website.name)],
```

Comment:
Since you removed the "Content" item, shouldn't this cause the test to fail?

Reply:
Yes, good catch. It seems like I had the `links` and `expected in the wrong order. `arrayContaining` should be used on the array with all of the items to be matched while the part on the left should be the subset